torch/_dynamo/utils.py

```diff
@@ -812,6 +726,11 @@ def to_int_us(v: Optional[float]) -> Optional[int]:
     return None if v is None else int(v * 1_000_000)
 
 
+# Version field added to every log. Increment to make it easier to distinguish new
+# vs. old entries when you make a substantive change to how the logs are populated.
+LOG_FORMAT_VERSION = 2
```

Comment:
Note that the field already exists in the internal thrift definition. We're just not logging it yet. I supposed this should be 1 though.

Reply:
2 seems fine, thanks